gensim/models/ldamodel.py

```diff
@@ -1231,7 +1230,7 @@ def load(cls, fname, *args, **kwargs):
 
         # the same goes for dtype (except it was added later)
         if not hasattr(result, 'dtype'):
-            result.dtype = np.float64 # float64 was used before as default in numpy
+            result.dtype = np.float64  # float64 was used before as default in numpy
             logging.warning("dtype was not set, so using np.float64")
```

Comment:
A more concrete message please. When reading this warning, users will be left scratching their heads: set where? Why? What does this mean to me?

How about `"dtype not set in saved %s file %s, assuming np.float64" % (result.__class__.__name__, fname)`? 
And only log at INFO or even DEBUG level, since it's an expected state when loading an old model, nothing out of ordinary.

Question: isn't it better to infer the dtype from the loaded object? Can it ever happen that it's something else, not `np.float64`?

Reply:
Fixed message, decided info level suites better.

About inferring. Not clear how to do it. Infer from `LdaState.eta` and `LdaState.sstats`? But then we had test that their sum is `np.float64`, so it's safe to assume that we don't loose precision when setting dtype to `np.float64` and `np.float32` is not enough.

Anyway, let's imagine situation some of `nd.arrays` are somehow of different dtype, like `np.float32` and some are `np.float64`. The right `dtype` is still `np.float64`.